plugins/EventGhost/__init__.py

```diff
@@ -632,18 +723,71 @@ def Configure(self, eventString="", waitTime=0):
             (waitTimeCtrl, 0, wx.ALL, 5),
             (panel.StaticText(text.text3), 0, wx.ALIGN_CENTER_VERTICAL),
         )
-        panel.sizer.AddMany(((sizer1, 0, wx.EXPAND), (sizer2, 0, wx.EXPAND)))
+
+        sizer3 = eg.HBoxSizer(
```

Comment:
optional: replace sizer3 and sizer4 with just ```        panel.AddLine(queueEventText, queueEventCtrl)
        panel.AddLine(restoreEventText, restoreEventCtrl)
```
It's a little shorter and the checkboxes are aligned.

Reply:
I do agree with you on this. and this is one of those things i really really wanted to fix. along with the eg.EqualizeWidths. But i do have a nice formatter that I use in all of my plugins that I have found does a fantastic job of lining things up. I am not a fan of the AddLine because It leaves a rather large gap between the lines. I just wanted to keep it the same as it had already been coded.